### PR TITLE
Get shopping cart articles in v1.1, pass custom auth to resolvers.py

### PR DESCRIPTION
**Pull request by DanielMaly**
Title says it all. The shopping cart articles request was completely missing from the v1.1 map and sometimes we need to work with authentication params less awkwardly than through environment variables.

### CODE REVIEW

File: mkmsdk/api_map.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# flake8: noqa
```
Comment (silvanocerza) [inline mkmsdk/api_map.py]: Given the nature of this file I prefer if we keep this flag so flake8 will skip it, otherwise it will show lots of errors.

**Comment by silvanocerza** [comment]
Hi Daniel, the PR looks good apart from that little issue with the api map file.
If you could add some tests it would be perfect. :smile:

File: mkmsdk/api_map.py (hunk 1 above)
Comment (DanielMaly) [reply mkmsdk/api_map.py]: Interesting, I didn't even know this line was there and I certainly didn't mean to remove it. I'll add it back.

**Comment by DanielMaly** [comment]
I'm sorry but I just don't have the time to write tests for this, especially since the new auth parameters would require some changes to how the tests are set up. All I can say is that the code has been running in production for almost a year now with no issues...

**Comment by silvanocerza** [comment]
Nevermind, I'll merge it anyway and write the tests myself later. :)

**Comment by silvanocerza** [comment]
I rebased the PR to keep a cleaner history, consider it merged.

**Comment by DanielMaly** [comment]
Is it? I'd love to finally switch from my own fork but I can't see the new code in master yet :)

**Comment by DanielMaly** [comment]
Nevermind, I just saw it in devel. I'll just wait for a new version release then.

**Comment by silvanocerza** [comment]
@DanielMaly Just letting you know I published the new version. :)

**Pull request closed without merging**